fineract-provider/src/main/java/org/apache/fineract/portfolio/floatingrates/api/FloatingRatesApiResource.java

```diff
@@ -45,7 +45,11 @@
 @Path("/floatingrates")
 @Component
 @Scope("singleton")
-@Api(value = "Floating Rates", description = "It lets you create, list, retrieve and upload the floating rates")
+@Api(tags = {"Floating Rates"})
+@SwaggerDefinition(tags = {
+		@Tag(name = "Floating Rates", description = "It lets you create, list, retrieve and upload the floating rates")
```

Comment:
Indentation of @Tag is not correct, please correct!

Reply:
From swagger's API documentation, here is the way to add description 
@Api(tags = {"Swagger Resource"})
@SwaggerDefinition(tags = {
    @Tag(name = "Swagger Resource", description = "Write description here")
})
public class ... {
}

does that mean i will have to unindent? I have not checked if that will work but right now, that description properly appears on the UI

Reply:
I'm seeing double ```tab-blank``` spaces in annotation 
```@Tag(name = "Floating Rates", description = "It lets you create, list, retrieve and upload the floating rates")```

Reply:
@Sanyam96 I would not worry too much about this kind of thing (whitespace and indent), we're about to have a general solution for this, please watch [FINERACT-821](https://issues.apache.org/jira/browse/FINERACT-821). (Help from you on that one would be very welcome BTW!)